docs/source/getting_started/index.ipynb

```diff
@@ -1139,6 +1159,54 @@
     "tbl.table.head(2)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "762799a5-5b99-46ca-8b68-401dfdf10100",
+   "metadata": {},
+   "source": [
+    "To include multiple years of data, you can include the years in the \"year\" parameter in the form of [Start Year, Stop Year]\n",
+    "\n",
+    "This also works in a dataset with only a single year of data"
```

Comment:
I think that you can drop the 2nd sentence

Reply:
I'd be interested to see how the code would run if I were to query a single year dataset using a multi-year input. I notice that if I include a year that does not exist in a multi-year dataset (say 2021-2027), the output will still be from 2021-2024, without any warning on the non-exist years. So I assumed that this would work on a single year dataset?

I might need to find a single year dataset and observe its output